projects/schematics/src/add-spartacus/spartacus-features.ts

```diff
@@ -147,6 +148,7 @@ function configureSpartacusModules(
                 'PaymentMethodsModule',
                 'NotificationPreferenceModule',
                 'MyInterestsModule',
+                'MyAccountV2NavigationModule',
```

Comment:
[MEDIUM]
As discussed offline, let's import in the `spartacus-features.module` (in our repo and in customers' repos via schematics) the (new) module named `MyAccountV2Module` which would encapsulate any shared my-account-v2 stuff (that doesn't belong to any feature library), e.g. it would import inside the `MyAccountV2NavigationModule`.

I'm aware that now `MyAccountV2Module` would import only `MyAccountV2NavigationModule` inside, so the introducing the parent module might seen redundant. However, the benefit of it is that the parent module (`MyAccountV2Module`) allows us in the future to add underneeth more modules, if we need to add  in the upcoming iterations of development of the My Account V2.

Reply:
done